### ISSUE TITLE
Model Schema not showing

### ISSUE THREAD

**Issue by lawtya** [swagger-api/swagger-ui#251]
Hi - hope you can help.
We have an implementation of swagger similar to swagger-springmvc-example.  We are running into issues when using swagger-ui when dealing with maps - see following as an example:
    @ApiOperation(value="Creates a compound case")
    @RequestMapping(value = "/v1/compoundCases", method = RequestMethod.PUT, consumes = "application/json", produces = "application/json; charset=utf-8")
    @ResponseStatus(HttpStatus.CREATED)
    @ResponseBody
    public Map<String, Object> createCompoundCase(@RequestBody Map<String, String> requestBody,  
            @ApiParam(value = "user token") @RequestParam("token") String token) throws Exception {  
        String caseId = requestBody.get("caseId");
        Case createdCase = userNuixEnvironmentManager.getUserNuixEnvironment(token).createCompoundCase(caseId);
        return caseService.convertCaseToMap(createdCase);
    }
    @ApiOperation(value="Creates a compound case")
    @RequestMapping(value = "/v1/compoundCases", method = RequestMethod.PUT, consumes = "application/json", produces = "application/json; charset=utf-8")
    @ResponseStatus(HttpStatus.CREATED)
    @ResponseBody
    public Map<String, Object> createCompoundCase(@RequestBody Map<String, String> requestBody,  
            @ApiParam(value = "user token") @RequestParam("token") String token) throws Exception {  
        String caseId = requestBody.get("caseId");
        Case createdCase = userNuixEnvironmentManager.getUserNuixEnvironment(token).createCompoundCase(caseId);
        return caseService.convertCaseToMap(createdCase);
    }
    @ApiOperation(value="Creates a compound case")
    @RequestMapping(value = "/v1/compoundCases", method = RequestMethod.PUT, consumes = "application/json", produces = "application/json; charset=utf-8")
    @ResponseStatus(HttpStatus.CREATED)
    @ResponseBody
    public Map<String, Object> createCompoundCase(@RequestBody Map<String, String> requestBody,  
            @ApiParam(value = "user token") @RequestParam("token") String token) throws Exception {  
        String caseId = requestBody.get("caseId");
        Case createdCase = userNuixEnvironmentManager.getUserNuixEnvironment(token).createCompoundCase(caseId);
        return caseService.convertCaseToMap(createdCase);
    }

Basically when looking at the model schema - its empty:

![results](https://f.cloud.github.com/assets/4869554/743480/8c9554f6-e3e8-11e2-82da-5f9629064143.png)

Ive seen a few references stating that maps are not supported - or that there is a problem with swagger.js but nothing defining what the problem actually is and whether there is a workaround.  Our restful api uses maps for the majority - so it would be great to get this sorted.

Can anyone help?

Thanks
